andela/ah-backend-realers
# #166187858 Users should be able to create articles

#### What does this PR do?

Enables logged in users to create Articles 

#### Description of task completed
* Created models file, serializers file, and view file with `article class`, `ArticleSerializer Class`, and `ArticleView class respectively` 
* Wrote tests for creating an article

#### How should this be manually tested on your local machine
* Make sure you [postman](https://www.getpostman.com/downloads/) installed on your local machine
* Do a `git fetch origin ft-create-articles-166187858`
* Do a `git checkout ft-create-articles-166187858`
* Run this command `$ python manage.py runserver`
* The url provided in your terminal should be tested on postman with the prefixes below
 `/api/users/` for `POST` method when the user is registering.

use the format below
`{
       "user":{
              "email": "email",
              "username": "username",
              "password": "password"
         }
               }`
* Then login on postman with the prefixes below
`/api/users/login/` for `POST` method.

use the format below
`{
       "user":{
              "email": "email",
               "password": "password"
         }
               }`
* After logging in, you should have received the token
* Create an article using postman with the prefixes below with the token in the authorization header
`/api/articles/ on a `POST` method. use the format below

`{
	"article":{
		"title": "title",
		"description": "description",
		"body": "body"
	}
}`


finishes [#166187858](https://www.pivotaltracker.com/n/projects/2350340/stories/166187858)
